articles/design/using-auth0-with-multi-tenant-apps.md

```diff
@@ -9,50 +9,48 @@ contentType: concept
 useCase: strategize
 ---
 
-# Using Auth0 with Multi-Tenant Applications
+# Using Auth0 to Secure Your Multi-Tenant Applications
 
-In this article, we'll discuss (at a high-level) how Auth0 can help you manage users for your multi-tenant applications.
+This article provides a high-level overview of how Auth0 can help you to secure your multi-tenant applications.

```

Comment:
This article provides a high-level overview of how Auth0 can help you ~to~ secure your multi-tenant applications.